articles/130_ros_time.md

```diff
@@ -140,6 +144,15 @@ The underlying datatypes will also provide ways to register notifications, howev
   <b>TODO:</b> Enumerate the <code>rcl</code> datastructures and methods here.
 </div>
 
+## Work Needing To Be Done (Apr 2017)
+
+- Decide where and when to autosubscribe to the `/clock` message. Built in to Node? A Node addon or service? Part of the first call to `rcl_node_init`?
```

Comment:
As I mentioned above I think that the "ROS Time" object will need to be created from the Node or by getting a node in from the constructor.

The "System" time object and the "Steady" time object can be independent of the node. The only reason to still have a custom version of those, rather than use `std::chrono` directly, is to present a similar enough interface to the "ROS Time" object so that other classes like Timers can use any of them interchangeably.